From 490869320ce139b823541b7699f2f2a16bc54dea Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 20 May 2018 08:48:49 -0700
Subject: [PATCH] Cross compile analyzer

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 analyzer/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/analyzer/Makefile b/analyzer/Makefile
index d5a9fb5..f90e4ff 100644
--- a/analyzer/Makefile
+++ b/analyzer/Makefile
@@ -27,7 +27,7 @@ JSONFILE=pgns.json
 all: $(TARGETS)

 $(ANALYZER): analyzer.c pgn.c analyzer.h pgn.h ../common/common.c ../common/common.h Makefile
-       $(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(ANALYZER) -I../common pgn.c analyzer.c ../common/common.c $(LDLIBS$(LDLIBS-$(@)))
+       $(BUILD_CC) $(BUILD_CPPFLAGS) $(BUILD_CFLAGS) $(BUILD_LDFLAGS) -o $(ANALYZER) -I../common pgn.c analyzer.c ../common/common.c $(LDLIBS$(LDLIBS-$(@)))

 json: $(ANALYZER) pgns2json.xslt
        $(ANALYZER) -explain-xml >$(XMLFILE) && xsltproc pgns2json.xslt $(XMLFILE) >$(JSONFILE)
--
2.17.0
