From da97fd791d65191bc214ec043b682c5a43c9d95a Mon Sep 17 00:00:00 2001
From: Shan Desai <des@biba.uni-bremen.de>
Date: Tue, 11 Jun 2019 11:53:44 +0200
Subject: [PATCH] Add non-root user to new Makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e4aa50d..7fc55d4 100755
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ clean:
 	
 install: rel/$(PLATFORM)/analyzer $(DESTDIR)$(BINDIR) $(DESTDIR)$(CONFDIR)
 	for i in rel/$(PLATFORM)/* util/* */*_monitor; do f=`basename $$i`; echo $$f; rm -f $(DESTDIR)$(BINDIR)/$$f; cp $$i $(DESTDIR)$(BINDIR); done
-	for i in config/*; do install -g $(ROOT_GID) -o $(ROOT_UID) -m $(ROOT_MOD) $$i $(DESTDIR)$(CONFDIR); done
+	for i in config/*; do install -m 0644 $$i $(DESTDIR)$(CONFDIR); done
 	-killall -9 actisense-serial ikonvert-serial n2kd socketcan-writer || echo 'No running processes killed'
 
 zip:
-- 
2.11.0

